scripts/lib/aws/sagemaker.sh

```diff
@@ -195,9 +197,150 @@ assert_aws_sagemaker_model_arn() {
   local model_response=$(daws sagemaker describe-model --model-name "$model_name" --region "$AWS_REGION")
   local aws_model_arn=$(echo "$model_response" | jq -r '.ModelArn')
   if [[ ( -z "$aws_model_arn" || "$expected_model_arn" != "$aws_model_arn" ) ]]; then
-    debug_msg "ERROR: ModelArn did not match/not found for $model_name"
+    debug_msg "[ERROR] ModelArn did not match/not found for $model_name"
     return 1
   fi
 
   return 0
-}
\ No newline at end of file
+}
+
+# is_aws_sagemaker_trainingjob_exists() returns 0 if an TrainingJob with the supplied name
+# exists, 1 otherwise.
+#
+# training_job_created TRAINING_JOB_NAME
+# Arguments:
+#
+#   TRAINING_JOB_NAME  required string for the name of the bucket to check
+#
+# Usage:
+#
+#   if ! training_job_created "$training_job_name"; then
+#       echo "Training Job $training_job_name does not exist!"
+#   fi
+is_aws_sagemaker_trainingjob_exists() {
+  if [[ "$#" -lt 1 || -z ${AWS_REGION+x} ]]; then
+    echo "[FAIL] Usage: is_aws_sagemaker_trainingjob_exists training_job_name. Expects AWS_REGION to be set"
+    exit 1
+  fi
+  local __training_job_name="$1"
+
+  local __training_job_status="$(daws sagemaker describe-training-job --region "$AWS_REGION" --training-job-name $__training_job_name --output json | jq .TrainingJobStatus)"
+
+  if [[ $? -eq 0 ]]; then
+      echo "$__training_job_name found!"
```

Comment:
recommend not placing `echo` statements in functions that just return a 0 or 1. Instead, place `echo` statements where you *call* these functions, if at all.